packages/material-ui/src/Chip/Chip.test.js

```diff
@@ -30,7 +30,6 @@ describe('<Chip />', () => {
       assert.strictEqual(wrapper.hasClass(classes.root), true);
       assert.strictEqual(wrapper.hasClass('my-Chip'), true);
       assert.strictEqual(wrapper.props()['data-my-prop'], 'woofChip');
-      assert.strictEqual(wrapper.props().tabIndex, -1);
```

Comment:
Probably better to explicitly assert the expected `tabIndex` instead of allowing anything.